app/models/recipient.rb

```diff
@@ -7,6 +7,10 @@ def notify! twilio = Message::SendNotificationTextMessage.new
     target_number = person.phone
     target_number = "+1#{person.phone}" if /\A\d{10}/.match target_number
     msg = twilio.sms_send notification.subject, target_number
-
+    if msg.is_a? Twilio::REST::RequestError
+      messages.create(status: "error", body: msg.message)
+    else
+      messages.create(status: "sent", body: msg)
```

Comment:
Should the body of the Message be the same in both cases ? ( `msg` vs `msg.message`).

Reply:
I was wrongly treating the success path as getting a string for `msg`. Looking at the API docs you linked earlier (https://www.twilio.com/docs/api/messaging/message) I see that a successful call to the API will return an object with a `body` attribute. For the error, though, the twilio gem converts the response into a subclass of `StandardError`, which has no `body` only a `message`.